src/core/packet_builder.c

```diff
@@ -822,13 +828,10 @@ QuicPacketBuilderFinalize(
         Builder->Metadata->PacketNumber,
         QuicPacketTraceType(Builder->Metadata),
         Builder->Metadata->PacketLength);
-    if (QUIC_FAILED(
-        QuicLossDetectionOnPacketSent(
-            &Connection->LossDetection,
-            Builder->Path,
-            Builder->Metadata))) {
-        goto Exit;
-    }
+    QuicLossDetectionOnPacketSent(
```

Comment:
Why shouldn't we exit when this fails an allocation internally?

Reply:
Opened this issue to track the bug https://github.com/microsoft/msquic/issues/1570